### PR TITLE
Escape html places

### PR DESCRIPTION
**Pull request by cmdcolin**
@garrettjstevens This is sort of a modification to to the escaping/unescaping approach that was going on.

If you want to test and give feedback let me know!

### CODE REVIEW

File: src/JBrowse/View/Track/_FeatureDetailMixin.js [hunk 1]
```diff
@@ -159,7 +159,7 @@ return declare( FeatureDescriptionMixin, {
                 },
                 container )
             additionalTags.sort().forEach( t => {
-                this.renderDetailField( atElement, t, f.get(t), f, undefined, track.store.getTagMetadata(t))
+                this.renderDetailField( atElement, t, Util.escapeHTML(f.get(t)), f, undefined, track.store.getTagMetadata(t))
```
Comment (garrettjstevens) [inline src/JBrowse/View/Track/_FeatureDetailMixin.js]: `f.get(t)` here can be an object, so escaping it is causing errors when I try to view details for a variant.

Comment (cmdcolin) [reply src/JBrowse/View/Track/_FeatureDetailMixin.js]: yep...pretty obvious miss there

**Comment by garrettjstevens** [comment]
@cmdcolin the idea of escaping by default unless one of those configs is present. I think I found a couple places where unescaped HTML might be coming through, though. It looks like it's probably happening in the mouseovers for CanvasFeatures and the description for HTMLFeatures, since the \<CN0\> and \<CN2\> are missing:
screenshot from 2018-11-24 07-56-06

**Comment by garrettjstevens** [comment]
Looks great to me! I think React just assumes that you need to escape everything. If you try to use unescaped text it triggers an eslint rule (see below). So with JB2 we might just need to say that everything is going to be interpreted as HTML so it should already be escaped. Plus with JSX syntax turned on you pretty much have to escape things like \< and \> or else you'll get syntax errors.
image

**Pull request merged**